x-pack/test/ui_capabilities/security_and_spaces/tests/catalogue.ts

```diff
@@ -13,7 +13,15 @@ import { UserAtSpaceScenarios } from '../scenarios';
 export default function catalogueTests({ getService }: FtrProviderContext) {
   const uiCapabilitiesService: UICapabilitiesService = getService('uiCapabilities');
 
-  const esFeatureExceptions = ['security', 'rollup_jobs', 'reporting', 'transform', 'watcher'];
+  const esFeatureExceptions = [
```

Comment:
__question:__ just for my own education, is this a list of features that depend on specific ES cluster privileges?

Reply:
Yes, these are what I'm calling "Elasticsearch Features", which are governed by cluster and/or index privileges as opposed to Kibana Privileges. This was a concept that we introduced in https://github.com/elastic/kibana/pull/67791 in order to know when it was safe to show/hide features within the Stack Management application.

Reply:
Thanks!